assets/css/addPost.css

```
@@ -0,0 +1,109 @@
+.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: none;
+    box-shadow: none;
+}
+
+
+#addPost-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--feedBackgroundColor);
+    border-radius: 15px;
+    width: 400px;
+    padding: 20px;
+    margin: 40px auto;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    margin: 40px auto;
+}
+
+.post {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
```

```suggestion
.post {
    display: flex;
    flex-direction: column;
    align-items: center;
    width: 100%;
}
```
Remove the unneccessary white background color and add the center alignment so that the button would be centered.